### DISCUSSION TITLE
Header Overlay on Image Buttons don't appear

### DISCUSSION THREAD

**Discussion by kfpro** [mmistakes/minimal-mistakes#2688] [Q&A] [answered]
The header overlay buttons don't appear. Added everything as instructed in the docs but seems to be a bug.
I'm using Github Pages and I am new to Jekyll.

from the markdown file:
---
title: "About"
permalink: /about/
date: 2018-01-28
tags: [data wrangling, data science, messy data]
excerpt: "Image : Gallery"
header:
  overlay_image: /images/ecopark_small.jpg
  overlay_filter: 0 # same as adding an opacity of 0.5 to a black background
  caption: "Photo credit: [**Gallery**](https://kfpro.github.io//gallery/)"
  actions:
    - label: "Foo Button"
      url: "https://kfpro.github.io//about/"
    - label: "Bar Button"
      url: "https://kfpro.github.io//about/"

layout: archive
author_profile: true
last_modified_at: 2019-08-27T15:46:43-04:00
toc: true
mathjax: "true"
---

From the generated website:

[My page about](https://kfpro.github.io//about/)

A screenshot: 

[Bug Screenshot](https://drive.google.com/file/d/1x98UvqIcN0K0gVcG06XVmKOZLzh3LbMg/view?usp=sharing)

## Environment

- Minimal Mistakes version:
- Ruby gem or remote theme version:
- Jekyll version:
- Git repository URL:
- GitHub Pages hosted (if yes provide URL to site):
- Operating system:

**Comment by mmistakes** [comment, marked as answer]
Not a bug.
You're using a really old version of the theme... `4.9.0`. This version doesn't have support for adding the button links in the header.

Upgrade to the lastest version `4.20.2`.

**Comment by kfpro** [comment]
Sure , Thanks for the prompt reply, really appreciate it !
